src/btree/bt_debug.c

```diff
@@ -757,7 +761,12 @@ __wt_debug_cursor_hs(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor)
     WT_ERR(hs_cursor->get_key(hs_cursor, &hs_btree_id, hs_key, &start.timestamp, &start.txnid,
       &stop.timestamp, &stop.txnid));
 
-    WT_ERR(__debug_time_pairs(ds, "T", start.timestamp, start.txnid, stop.timestamp, stop.txnid));
+    /*
+     * FIXME: Currently, this function is not called anywhere in the code. In future if it is used,
+     * it needs some fixes related to accessing the proper key and value of history store.
+     */
+    // WT_ERR(__debug_time_pairs(ds, "T", start.timestamp, start.txnid, stop.timestamp,
+    // stop.txnid));
```

Comment:
Could you explain why this isn't working now?
The comment you've used to disable the code isn't part of our coding standard.

Reply:
The retrieval of the history store key/value are not updated to the latest format. I fixed them now and removed the FIXME and code commenting.